docs/DeveloperGuide.md

```diff
@@ -329,9 +329,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `17` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+1.  The system should be able to store data related to university students, including personal and contact information. (Data)
```

Comment:
I think this falls under functional requirements. Could we consider scrapping this as it is not NFR?